lib/api/core/models/repositories/roleRepository.js

```diff
@@ -130,27 +130,20 @@ RoleRepository.prototype.loadRole = function roleRepositoryLoadRole (roleId) {
 /**
  *
  * @param {Array} controllers
- * @param {int} from
- * @param {int} size
  */
-RoleRepository.prototype.searchRole = function roleRepositorySearchRole (controllers, from, size) {
-  var query = {};
-
-  if (controllers && Array.isArray(controllers)) {
-    query.bool = {should: []};
-
-    controllers.forEach(controller => {
-      query.bool.should.push({exists: {field : 'controllers.' + controller}});
+RoleRepository.prototype.searchRole = function roleRepositorySearchRole (controllers) {
```

Comment:
In any case, we have to fetch all roles from ES, so if we want pagination, we need to sort the results and then return an subset according to pagination arguments.

In term of performances, returning a paginated content will not be faster than returning all the contents.

But I'm ok to re-add it.